ee/modules/160-multitenancy-manager/docs/USAGE.md

```diff
@@ -0,0 +1,255 @@
+      ---
+      # Max requests and limits for resource consumption per pod in namespace.
+      # All containers in a namespace must have requests and limits.
+      # Refer to https://kubernetes.io/docs/concepts/policy/limit-range/
+      apiVersion: v1
+      kind: LimitRange
+      metadata:
+        name: all-containers
+      spec:
+        limits:
+          - max:
+              {{ with .params.limits.cpu }}cpu: {{ . }}{{ end }}
+              {{ with .params.limits.memory }}memory: {{ . }}{{ end }}
+            maxLimitRequestRatio:
+              cpu: 1
+              memory: 1
+            type: Container
+      ---
+      # Deny all network traffic by default except namespaced traffic and dns.
+      # Refer to https://kubernetes.io/docs/concepts/services-networking/network-policies/
+      kind: NetworkPolicy
+      apiVersion: networking.k8s.io/v1
+      metadata:
+        name: deny-all-except-current-namespace
+      spec:
+        podSelector:
+          matchLabels: {}
+        policyTypes:
+          - Ingress
+          - Egress
+        ingress:
+          - from:
+              - namespaceSelector:
+                  matchLabels:
+                    kubernetes.io/metadata.name: "{{ .projectName }}"
+        egress:
+          - to:
+              - namespaceSelector:
+                  matchLabels:
+                    kubernetes.io/metadata.name: "{{ .projectName }}"
+          - to:
+              - namespaceSelector:
+                  matchLabels:
+                    kubernetes.io/metadata.name: kube-system
+            ports:
+              - protocol: UDP
+                port: 53
+  ```
+  
+  ```shell
+  $ kubectl create -f project-type.yaml
+  projecttype.deckhouse.io/test-project-type created
+  
+  $ kubectl get projecttypes.deckhouse.io
+  NAME                READY   MESSAGE
+  test-project-type   true
+  ```
+
+- Create a [Project](cr.htlm#project) that contains the name of the previously created [ProjectType](cr.htlm#projecttype) in the [.spec.projectTypeName](cr.htlm#project-v1alpha1-spec-projecttypename) field. Fill in the [.spec.template](cr.htlm#project-v1alpha1-spec-template) field with values suitable for [.spec.openAPI ProjectType field](cr.htlm#projecttype-v1alpha1-spec-openapi):
+
+  ```yaml
+  # project.yaml
+  ---
+  apiVersion: deckhouse.io/v1alpha1
+  kind: Project
+  metadata:
+    name: test-project
+  spec:
+    description: Test case from Deckhouse documentation
+    projectTypeName: test-project-type
+    template:
+      requests:
+        cpu: 5
+        memory: 5Gi
+        storage: 1Gi
+      limits:
+        cpu: 5
+        memory: 5Gi
+  ```
+  
+  ```shell
+  $ kubectl create -f project.yaml
+  project.deckhouse.io/test-project created
+  
+  $ kubectl get projects.deckhouse.io
+  NAME           READY   DESCRIPTION                              MESSAGE
+  test-project   true    Test case from Deckhouse documentation
```

Comment:
There is a problem with the feedback. I created a project type with an invalid template, but in the cluster, I see:
```
NAME           READY   DESCRIPTION                              MESSAGE
test-project   true    Test case from Deckhouse documentation
```
However, Deckhouse output is 
```
Latest release 'd8-system/multitenancy-manager': revision: 1 has status: deployed
ModuleRun failed in phase 'CanRunHelm'. Requeue task to retry after delay. Failed count is 4. Error: helm upgrade failed: error validating "": error validating data: ValidationError(NetworkPolicy.spec.egress[1].ports[0]): unknown field "groups" in io.k8s.api.networking.v1.NetworkPolicyPort
```

I expect status to be not ready, but `Processing`, Creating` or `Deploying`, that Deckhouse saw the project resource and started doing something. The message should be something like `Deckhouse is creating the project, see deckhouse logs for more details`.